resources/dicts/events/misc_events/general/general.json

```diff
@@ -220,6 +220,46 @@
         "camp": "any",
         "tags": ["Newleaf", "Greenleaf", "Leaf-bare", "Leaf-fall", "other_cat_mate", "clan_kits", "to_both", "platonic", "romantic"],
         "event_text": "m_c and r_c watch the Clan's kits play and daydream about having their own one day."
+     },
+     {
+        "camp": "any",
+        "tags": ["Newleaf", "Greenleaf", "Leaf-bare", "Leaf-fall", "other_cat_mate"],
+        "event_text": "m_c watches r_c give an elder the best piece of prey on the pile. {PRONOUN/m_c/subject} feel lucky to have such a kind cat as their mate.",
+        "other_cat_trait": ["compassionate", "loving", "thoughtful"],
+        "negate_cat_trait": ["bloodthirsty", "cold", "fierce", "troublesome", "vengeful"],
+        "negate_other_cat_trait": ["bloodthirsty", "cold", "fierce", "troublesome", "vengeful"]
+     },
+     {
+        "camp": "any",
+        "tags": ["Newleaf", "other_cat_mate", "to_both", "platonic"],
+        "event_text": "m_c sits still as r_c helps remove some leftover Leaf-bare fluff from the back of {PRONOUN/m_c/poss} neck."
+     },
+     {
+        "camp": "any",
+        "tags": ["Newleaf", "Greenleaf", "Leaf-bare", "Leaf-fall", "other_cat_mate", "to_both", "platonic", "romantic", "clan_kits"],
+        "event_text": "m_c listens bashfully as r_c tells the story of how {PRONOUN/r_c/subject} fell in love with {PRONOUN/r_c/poss} mate to a group of starry-eyed kits."
+     },
+     {
+        "camp": "any",
+        "tags": ["Leaf-bare", "other_cat_mate", "to_both", "platonic", "romantic"],
+        "event_text": "m_c and r_c are so cuddled up and cozy in their nest they almost miss the dawn patrol."
+     },
+     {
+        "camp": "any",
+        "tags": ["Leaf-fall", "other_cat_mate", "to_both", "platonic", "romantic"],
+        "event_text": "r_c gives m_c the prettiest fallen leaf {PRONOUN/r_c/subject} can find. m_c wears it proudly.",
+        "accessories": ["MAPLE LEAF"]
+     },
+     {
+        "camp": "any",
+        "tags": ["Greenleaf", "other_cat_mate", "to_both", "platonic"],
+        "event_text": "m_c and r_c spend the evening chasing cicadas around camp. M_c keeps a trophy of {PRONOUN/m_c/poss} catch on {PRONOUN/m_c/poss} pelt.",
```

Comment:
all cat abbreviations (m_c, r_c) must be completely lowercase, even when beginning a sentence.  'catch' needs to be a VERB tag